src/core_plugins/timelion/public/directives/timelion_expression_suggestions/timelion_expression_suggestions.html

```diff
@@ -12,44 +12,61 @@
+    ng-repeat="suggestion in suggestions track by suggestion.name"
     ng-mousedown="onMouseDown($event)"
     ng-click="onClickSuggestion({ suggestionIndex: $index })"
     aria-label="{{suggestion.name}}"
     aria-describedby="timelionSuggestionDescription{{$index}}"
     data-test-subj="timelionSuggestionListItem"
   >
-    <h4>
-      <strong>.{{suggestion.name}}()</strong>
-      <small id="timelionSuggestionDescription{{$index}}">
-        {{suggestion.help}}
-        {{suggestion.chainable ? '(Chainable)' : '(Data Source)'}}
-      </small>
-    </h4>
 
-    <div ng-show="suggestion.args.length > (suggestion.chainable ? 1: 0)">
-      <div ng-show="suggestions.length > 1">
-        <strong>Arguments:</strong>
-        <span ng-repeat="arg in suggestion.args" ng-hide="$index < 1 && suggestion.chainable">
-          <strong>{{arg.name}}</strong>=(<em>{{arg.types.join(' | ')}}</em>)
-          <em ng-show="!$last">,</em>
-        </span>
+    <div ng-switch on="suggestionsType">
+
+      <div ng-switch-when="functions">
+        <h4>
+          <strong>.{{suggestion.name}}()</strong>
+          <small id="timelionSuggestionDescription{{$index}}">
+            {{suggestion.help}}
+            {{suggestion.chainable ? '(Chainable)' : '(Data Source)'}}
+          </small>
+        </h4>
+
+        <div ng-show="suggestion.args.length > (suggestion.chainable ? 1: 0)">
+          <div ng-show="suggestions.length > 1">
+            <strong>Arguments:</strong>
+            <span ng-repeat="arg in suggestion.args" ng-hide="$index < 1 && suggestion.chainable">
+              <strong>{{arg.name}}</strong>=(<em>{{arg.types.join(' | ')}}</em>)
+              <em ng-show="!$last">,</em>
+            </span>
+          </div>
+
+          <div class="suggestion-details" ng-show="suggestions.length === 1">
+            <table class="table table-striped table-condensed table-bordered">
+            <thead>
+              <th>Argument Name</th>
+              <th>Accepted Types</th>
+              <th>Information</th>
+            </thead>
+            <tr ng-repeat="arg in suggestion.args" ng-hide="$index < 1 && suggestion.chainable">
+              <td>{{arg.name}}</td>
+              <td><em>{{arg.types.join(', ')}}</em></td>
+              <td>{{arg.help}}</td>
+            </tr>
+            </table>
+          </div>
+        </div>
       </div>
 
-      <div class="suggestion-details" ng-show="suggestions.length === 1">
-        <table class="table table-striped table-condensed table-bordered">
-        <thead>
-          <th>Argument Name</th>
-          <th>Accepted Types</th>
-          <th>Information</th>
-        </thead>
-        <tr ng-repeat="arg in suggestion.args" ng-hide="$index < 1 && suggestion.chainable">
-          <td>{{arg.name}}</td>
-          <td><em>{{arg.types.join(', ')}}</em></td>
-          <td>{{arg.help}}</td>
-        </tr>
-        </table>
+      <div ng-switch-when="arguments">
```

Comment:
I think if we change this to:

```html
      <div ng-switch-when="arguments">
        <h4>
          <strong>{{suggestion.name}}=</strong>
          <small id="timelionSuggestionDescription{{$index}}">
            {{suggestion.help}}
          </small>
        </h4>
        <div>
          <strong>Accepts:</strong>
          <em>{{suggestion.types.join(', ')}}</em>
        </div>
      </div>
```

Then it will look a little more consistent with how the function suggestions appear:

![image](https://user-images.githubusercontent.com/1238659/32190918-946996b2-bd6c-11e7-87db-e19b283da5f9.png)